lib/tasks/mfa_policy.rake

```diff
@@ -0,0 +1,32 @@
+require "resolv"
+
+def mx_exists?(email)
+  domain = email.split("@").last
+  mx_resolver = Resolv::DNS.new
+  mx_resolver.timeouts = 10
+
+  return false if mx_resolver.getresources(domain, Resolv::DNS::Resource::IN::MX).empty?
+  true
+rescue StandardError => e
+  puts "Error during processing: #{$ERROR_INFO}"
+  puts "Backtrace:\n\t#{e.backtrace.join("\n\t")}"
+  false
+end
+
+namespace :mfa_policy do
```

Comment:
I decided to create a separate namespace instead of adding this task to `mfa_notification` because we foresee a few other MFA Policy-related tasks. So, it seemed reasonable to group those particular tasks together (e.g. we'll set up the reminder email here too - the one that we'll send before MFA enforcement goes into effect).